acceptable/__main__.py

```diff
@@ -264,53 +225,39 @@ def render_markdown(metadata, cli_args):
         'title': 'Index',
         'location': 'index.' + cli_args.extension,
     }]
-    api_groups = defaultdict(list)
-    deprecated = []
-    sort_key = itemgetter('title')
     version = metadata.pop('$version', None)
     changelog = defaultdict(dict)
 
-    for api_name, api in metadata.items():
-        if api.get('undocumented', False):
-            continue
-        page_file = '{}.{}'.format(api_name, cli_args.extension)
-        page = {'title': api_name, 'location': page_file}
-
-        if api.get('deprecated_at', False):
-            deprecated.append(page)
-        else:
-            api_groups[api.get('api_group')].append(page)
-
-        for changed_version, log in api['changelog'].items():
-            changelog[changed_version][api_name] = log
+    for group in metadata:
+        apis = metadata[group]['apis']
+        for api in apis.values():
+            # collect global changelog
+            for changed_version, log in api.get('changelog', {}).items():
+                changelog[changed_version][api['api_name']] = log
 
-        path = os.path.join('en', page_file)
-        yield path, cli_args.page_template.render(name=api_name, **api)
-
-    if len(api_groups) == 1:
-        # only one group, flat navigation
-        navigation.extend(
-            sorted(list(api_groups.values())[0], key=sort_key)
+        all_undocumented = all(
+            api.get('undocumented', False) for api in apis.values()
         )
-    else:
-        default_group = api_groups.pop(None, None)
-        if default_group is not None:
-            navigation.extend(
-                sorted(default_group, key=sort_key),
+
+        if not all_undocumented:
```

Comment:
Instead of checking for `not all undocumented` we could do `any documented` instead